packages/node-common/package.json

```diff
@@ -25,9 +25,14 @@
   },
   "dependencies": {
     "@cursorless/common": "workspace:*",
-    "@cursorless/cursorless-engine": "workspace:*",
     "glob": "^10.3.10",
-    "pathe": "^1.1.2"
+    "lodash-es": "^4.17.21"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@types/chai": "^4.3.14",
+    "@types/js-yaml": "^4.0.9",
+    "@types/lodash-es": "4.17.0",
+    "chai": "^5.1.0",
+    "js-yaml": "^4.1.0"
```

Comment:
how come these are dev deps?

Reply:
I just mirrored what is on main. chai is by definition a developer tool and not really in production, but how you use these two different dependence lists is a bit debatable
https://github.com/cursorless-dev/cursorless/blob/4282c4bb13a3a3f0953f9a91550e80dbb43cf83e/packages/common/package.json#L29

Reply:
i'm ok with it for now; we're not particularly principled with our dev / prod deps at this point anyway, so we can revisit in the future

Reply:
Sounds reasonable